lxd/api_cluster.go

```diff
@@ -46,14 +46,71 @@ func clusterGet(d *Daemon, r *http.Request) Response {
 		name = ""
 	}
 
+	memberConfig, err := clusterGetMemberConfig(d.cluster)
+	if err != nil {
+		return SmartError(err)
+	}
+
 	cluster := api.Cluster{
-		ServerName: name,
-		Enabled:    name != "",
+		ServerName:   name,
+		Enabled:      name != "",
+		MemberConfig: memberConfig,
 	}
 
 	return SyncResponseETag(true, cluster, cluster)
 }
 
+// Fetch information about all node-specific configuration keys set on the
+// storage pools and networks of this cluster.
+func clusterGetMemberConfig(cluster *db.Cluster) ([]api.ClusterMemberConfigKey, error) {
+	var pools map[string]map[string]string
+	var networks map[string]map[string]string
+
+	keys := []api.ClusterMemberConfigKey{}
+
+	err := cluster.Transaction(func(tx *db.ClusterTx) error {
+		var err error
+		pools, err = tx.StoragePoolsNodeConfig()
+		if err != nil {
+			return errors.Wrapf(err, "failed to fetch storage pools configuration")
+		}
+		networks, err = tx.NetworksNodeConfig()
+		if err != nil {
+			return errors.Wrapf(err, "failed to fetch networks configuration")
```

Comment:
Errors must start with a capital (and never end with a period)

Reply:
Fixed.